src/libraries/Native/Unix/System.IO.Compression.Native/CMakeLists.txt

```diff
@@ -14,7 +14,6 @@ set(NATIVECOMPRESSION_SOURCES
 )
 
 if (NOT CLR_CMAKE_TARGET_BROWSER)
```

Comment:
Could you please also delete the unused methods from `src\libraries\Native\Windows\System.IO.Compression.Native\System.IO.Compression.Native.def` ?

Reply:
Looks like there are also actual `BROTLI_DEC_API`/`BROTLI_ENC_API` which we are not using.

Ex:
`BROTLI_DEC_API BROTLI_BOOL BrotliDecoderHasMoreOutput(...)`
`BROTLI_ENC_API size_t BrotliEncoderMaxCompressedSize(size_t input_size);`

I will assume that in our use patterns we have no need for these and will remove the decorations.

Reply:
Or better - replace with some noop like `BROTLI_INTERNAL`, - so that we could find them easily if needed

Reply:
Changing upstream code is not ideal, we have tried so far to keep them (brotli, zlib, libunwind and rapidjson) drop-in for next version. Maybe we can configure fvisibility and friends to achieve the desired result? Or strip those symbols after target is built.

Reply:
It is tempting to set exports externally via `version-scripts`. That is known for causing portability issues in the past, but maybe, since this is a simple leaf dll case, it would work here.

Reply:
I made the Unix exports cpecified externally via version-scripts. We will set `BROTLI_SHARED_COMPILATION` so that all stuff intended to be public is public, as before. 
But then what is actually exported will be reduced to what we use by passing explicit list of exports to the linker.

Reply:
So, now `System.IO.Compression.Native` exports exactly the number entries that is used by managed part. That is on all platforms - Windows, Unix, OSX, IOS. 
Perhaps that will allow for some bits to be trimmed off.